src/utils.ts

```diff
@@ -344,6 +347,30 @@ export const PotionMap: { [k in Potion]: { name: string, order: number, image: S
       def: Math.floor(5 + (skills.def * 0.15)),
     }),
   },
+  [Potion.RUBY_HARVEST]: {
+    name: 'Ruby Harvest',
+    order: 18,
+    image: RubyHarvest,
+    calculateFn: (skills) => ({
+      atk: Math.floor(4 + (skills.atk * 0.15)),
```

Comment:
nit: this should be `trunc` not `floor` (I'm aware the other ones are wrong too)